Spec/Auth.swift

```diff
@@ -1801,6 +1801,55 @@ class Auth : QuickSpec {
                 }
             }
 
+            // RSA9h
+            it("should use configured defaults if the object arguments are omitted") {
+                let options = AblyTests.commonAppSetup()
+                let rest = ARTRest(options: options)
+
+                let tokenParams = ARTTokenParams()
+                tokenParams.clientId = "tester"
+                tokenParams.ttl = 2000
+                tokenParams.capability = "{\"foo:*\":[\"publish\"]}"
+
+                let authOptions = ARTAuthOptions()
+                authOptions.queryTime = true
+                authOptions.key = options.key
+
+                var serverTimeRequestCount = 0
+                let hook = rest.testSuite_injectIntoMethodAfter(#selector(rest.time(_:))) {
+                    serverTimeRequestCount += 1
+                }
+                defer { hook.remove() }
+
+                waitUntil(timeout: testTimeout) { done in
+                    rest.auth.createTokenRequest(tokenParams, options: authOptions) { tokenRequest, error in
+                        expect(error).to(beNil())
+                        guard let tokenRequest = tokenRequest else {
+                            XCTFail("TokenRequest is nil"); done(); return
+                        }
+                        expect(tokenRequest.clientId) == tokenParams.clientId
+                        expect(tokenRequest.ttl) == tokenParams.ttl
+                        expect(tokenRequest.capability) == tokenParams.capability
+                        done()
+                    }
+                }
+
+                waitUntil(timeout: testTimeout) { done in
+                    rest.auth.createTokenRequest { tokenRequest, error in
```

Comment:
I am not sure this is right.  Unlike `authorize`, the defaults are not stored for subsequent requests, yet with this test it appears you assume they are?